scripts/format-queries.lua

```diff
@@ -106,25 +106,19 @@ local format_queries = [[
 ;; Only starts indent if 2 or more elements
 (list
   "[" @format.indent.begin
-  .
-  (_)
-  .
-  (_)
   "]" @format.indent.dedent)
 ;; Otherwise, remove brackets
 (list
-  "[" @format.remove
+  "[" @format.remove @format.cancel-append
   .
   (_) @format.cancel-append
   .
   "]" @format.remove)
 ;; [ ... ] @capture1 @capture2
-(list
-  (capture) @format.prepend-space)
 ;; Append newlines for nodes inside the list
 (list
   (_) @format.append-newline
-  (#not-has-type? @format.append-newline capture))
+  (#not-has-type? @format.append-newline capture quantifier))
```

Comment:
Just to be clear, this line is responsible for the `?` and `*` after a list to add a newline. Oversight happened